tools/Environments/DevHome.Environments/ViewModels/OperationsViewModel.cs

```diff
@@ -98,6 +113,38 @@ public void InvokeAction()
         });
     }
 
+    [RelayCommand]
+    public void InvokeAction()
+    {
+        // Show confirmation popup in case of delete
+        if (ComputeSystemOperation == ComputeSystemOperations.Delete)
+        {
+            ContentDialog noWifiDialog = new ContentDialog
+            {
+                Title = _stringResource.GetLocalized("DeleteEnviroment_Title"),
+                Content = _stringResource.GetLocalized("DeleteEnviroment_Content"),
+                PrimaryButtonText = _stringResource.GetLocalized("DeleteEnviroment_ConfirmButton"),
+                SecondaryButtonText = _stringResource.GetLocalized("DeleteEnviroment_CancelButton"),
+                XamlRoot = _windowEx?.Content.XamlRoot,

```

Comment:
I believe this will crash if XamlRoot is null.

Reply:
It won't run if it is. See the following lines.